src/objects.c

```diff
@@ -109,11 +109,11 @@ s8 D_800DC820[10] = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
 
 s8 D_800DC840[8] = { 9, 1, 2, 3, 4, 5, 7, 0 };
 
-s8 D_800DC848 = 0;
-u32 D_800DC84C[3] = {
-    0xFF401000,
-    0x1040FF00,
-    0x10FF4000,
+s8 gNoBoundsCheck = FALSE;
+u32 gMagnetColours[3] = {
+    ((255 << 24) | (64 << 16) |  (16 << 8) | 0), // Level 1
```

Comment:
Not at a comp to check, but can this use the macro for rgba 5551? Something like Gbapack_5551 or whatever it's called?

Reply:
It's 32 bit colour, but I can probably just do another macro for this for readability reasons